### PR TITLE
INTYGFV-16641: Refactering of integration tests for easier usage. Add…

### PR DESCRIPTION
**Pull request by mikeyOne**
…ed tests for fk3226 and fk7809.

### CODE REVIEW

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7809/CertificateModelFactoryFK7809.java [hunk 1]
```diff
@@ -71,7 +71,7 @@
 public class CertificateModelFactoryFK7809 implements CertificateModelFactory {
 
   private static final String GENERAL_LABEL_FUNKTIONSNEDSATTNING = "Beskriv funktionsnedsättningen, om möjligt med grad. Ange även eventuella undersökningsfynd.";
-  @Value("${certificate.model.fk7210.v1_0.active.from}")
+  @Value("${certificate.model.fk7809.v1_0.active.from}")
```
Comment (ChrilleSigerhed) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7809/CertificateModelFactoryFK7809.java:74]: Ops! Nice catch! :D

File: certificate-service/app/src/testFixtures/java/se/inera/intyg/certificateservice/application/testdata/TestDataCommonUserDTO.java [hunk 2]
```diff
@@ -103,6 +117,26 @@ public static UserDTOBuilder alvaVardadministratorDtoBuilder() {
         .healthCareProfessionalLicence(Collections.emptyList());
   }
 
+  public static UserDTOBuilder annaSJukskoterskaDtoBuilder() {
```
Comment (ChrilleSigerhed) [inline certificate-service/app/src/testFixtures/java/se/inera/intyg/certificateservice/application/testdata/TestDataCommonUserDTO.java:120]: Maybe fix the capital "J"

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
Wow, excellent work with this. Now we have established a clear and good pattern for how we will implement integrationtests in the future, I really like what you did here!

I have one small request, and that is that we maybe could update the project structure and place the integrationtests in different packages so that they are more easily located. For example maybe tests regarding messages, like MessagesIT, should live in an folder called messages.

What do you think about that?

**Review by ChrilleSigerhed — APPROVED** [review]
Awsome work! Well done! Will handle filltype.MAXIMAL later as discussed.

File: certificate-service/app/src/testFixtures/java/se/inera/intyg/certificateservice/application/testdata/TestDataCommonUserDTO.java (hunk 2 above)
Comment (mikeyOne) [reply certificate-service/app/src/testFixtures/java/se/inera/intyg/certificateservice/application/testdata/TestDataCommonUserDTO.java]: done

**Pull request merged**